federation-js/src/composition/validate/preComposition/__tests__/externalUsedOnBase.test.ts

```diff
@@ -43,6 +45,12 @@ describe('externalUsedOnBase', () => {
       Array [
         Object {
           "code": "EXTERNAL_USED_ON_BASE",
+          "locations": Array [
```

Comment:
Can this location point to the `@external` directive on this field specifically?